sources/osg/CullSettings.js

```diff
@@ -1,34 +1,54 @@
 'use strict';
-var Vec3 = require( 'osg/Vec3' );
 
 
 var CullSettings = function () {
-    this._computeNearFar = true;
-    this._nearFarRatio = 0.005;
-
-    var lookVector = Vec3.createAndSet( 0.0, 0.0, -1.0 );
-    /*jshint bitwise: false */
-    this.bbCornerFar = ( lookVector[ 0 ] >= 0 ? 1 : 0 ) | ( lookVector[ 1 ] >= 0 ? 2 : 0 ) | ( lookVector[ 2 ] >= 0 ? 4 : 0 );
-    this.bbCornerNear = ( ~this.bbCornerFar ) & 7;
-    /*jshint bitwise: true */
-    this._enableFrustumCulling = false;
-
-    // who sets the parameter
-    // if it's cullvisitor
-    // it's an OVERRIDER for enableFrustumCulling
-    // allowing for global EnableFrustimCulling
-    this._settingsSourceOverrider = this;
-    //LOD bias for the CullVisitor to use
-    this._LODScale = 1.0;
+
+    // Not doing a this.reset()
+    // because of multiple inheritance
+    // it will call the wrong reset
+    // cullstack reset for isntance()
+    CullSettings.prototype.reset.apply( this );
```

Comment:
@Kuranes could you check this point ^ I think it comes from a google talk or best practice for performance